static/js/settings_data.js

```diff
@@ -1,7 +1,13 @@
 import {page_params} from "./page_params";
-import * as people from "./people";
 import * as settings_config from "./settings_config";
 
+let user_join_days;
+export function initialize(current_user) {
+    const current_datetime = new Date();
+    const person_date_joined = new Date(current_user.date_joined);
+    user_join_days = (current_datetime - person_date_joined) / 1000 / 86400;
```

Comment:
I think we want this function to not include the current time; that way, if the browser stays open for 5 days, we'll start producing incorrect answers.  What we want is to just have this function be passed the user's `date_joined` property )ideally converted to a date object nicely), and save that so that we can do calculations with it later.

Reply:
Done.